### ISSUE TITLE
Piwik driver

### ISSUE THREAD

**Issue by nlochschmidt** [metabase/metabase#4097]
It would be neat to have support for [Piwik](https://piwik.org/) similar to the way GA support has been added #3020.

**Comment by mavimo** [comment]
@nlochschmidt PiWik use MySQL as database, so you can just connect to it. See [requirements](http://piwik.org/docs/requirements/) and [database schema](https://developer.piwik.org/guides/persistence-and-the-mysql-backend)

**Comment by nlochschmidt** [comment]
@mavimo you are right, however we don't host Piwik ourselfes but use the managed version from https://cloud.piwik.pro
I'll be checking tomorrow to see if they let us connect directly to the database, though probably not.

**Comment by salsakran** [comment]
@nlochschmidt we're unlikely to work on this in the immediate future. 

If you're interested in working on it, check out https://github.com/metabase/metabase/wiki/Writing-A-Driver and see if piwik's API offers up the needed info to power a driver.
